test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -236,6 +238,9 @@ def convert_date_time(cls, dt=None, fmt=None):
         if dt is None:
             return None
 
+        offset = timedelta(seconds=-time.timezone)
```

Comment:
I'm not sure if I understand this change. time.timezone will return the offset of current Tz frm UTC, so for me it gives offset as 5:30:00. Which means that if time.daylight is not True, then the code will directly subtract offset.total_seconds() from the epoch time, so that will subtract 19800.0 seconds from the epoch time?
Shouldn't you be setting the offset to just (-time.altzone - time.timezone) ?